nx/lib/nx.ex

```diff
@@ -5234,21 +5250,12 @@ defmodule Nx do
   """
   @doc type: :element
   def logical_not(tensor) do
```

Comment:
@josevalim Should I add docs for all of the functions implemented in this PR, or should we just leave it as a given?
Or maybe just for sort, argsort, top_k and reflect, which can be a bit trickier?